android/walletlib/src/main/java/com/solana/mobilewalletadapter/walletlib/authorization/AccountRecordsSchema.java

```diff
@@ -0,0 +1,29 @@
+package com.solana.mobilewalletadapter.walletlib.authorization;
+
+/*package*/ interface AccountRecordsSchema {
+    String TABLE_ACCOUNTS = "accounts";
+    String COLUMN_ACCOUNTS_ID = "id"; // type: long
+    String COLUMN_ACCOUNTS_PUBLIC_KEY_RAW = "public_key_raw"; // type: byte[]
+    String COLUMN_ACCOUNTS_LABEL = "label"; // type: String
+    String COLUMN_ACCOUNTS_ICON = "icon"; // type: String
+    String COLUMN_ACCOUNTS_CHAINS = "chains"; // type: String
```

Comment:
I'm storing the chains and feature arrays as comma delimited strings. I know this is normally frowned upon but this saves some complexity. We do not need to query on these parameters so  it seemed an acceptable tradeoff.